chapters/functions.tex

```diff
@@ -1712,14 +1712,14 @@ \section{External Function Interface}\label{external-function-interface}
 This is just as for any other function.  The components in the protected part allow local variables for temporary storage to be declared.
 \end{nonnormative}
 
-The \lstinline!language-specification! must currently be one of \lstinline!"builtin"!, \lstinline!"C"!, \lstinline!"C..."! (for one of the specific C standards like C89, C99, and C11 -- specifying
+The \lstinline!language-specification! must currently be one of \lstinline!"builtin"! (deprecated), \lstinline!"C"!, \lstinline!"C..."! (for one of the specific C standards like C89, C99, and C11 -- specifying
 that it relies on the C standard library of that version) or \lstinline!"FORTRAN 77"!.  Unless the external language is specified, it is assumed to be \lstinline!"C"!.
 
 \begin{nonnormative}
 The intended use of e.g.\ C99 is to detect if the user tries to link with a C99-function using a C89 compiler.
 \end{nonnormative}
 
-The \lstinline!"builtin"! specification is only used for functions that are defined
+The deprecated \lstinline!"builtin"! specification is only used for functions that are defined
```

Comment:
I think it would be good to specify `"builtin"` as being a tool-specific options that does not offer portability between tools. Perhaps:
```latex
The \lstinline!"builtin"! language specification has implementation-defined results and has been deprecated in order to promote interoperability between tools.
\begin{nonnormative}
Existing code using the \lstinline!"builtin"! language specification typically mimics the built-in operator \Cref{ref-to-operators-section} with the same name and tools are recommended to continue supporting this for the time being.
\end{nonnormative}
```

Reply:
Wouldn't that introduce a different meaning at the same time as it is deprecated?  I mean, the original intention was never to have different results depending on implementation, right?  I find the next sentence in the text a pretty good explanation of what it is:
> The external-function call mechanism for `"builtin"` functions is implementation-defined.

Also, describing it as a tool-specific option make it sound as if `"builtin"` is what tools are expected to use when having a tool-specific external interface.  I think it makes more sense to use a non-standardized tool-specific name for the external language in this case, so that it is clear that the function is not expected to exist in other tools.

Having some non-normative text that explains a deprecated concept seems meaningful, but I don't think we should say something like this about every deprecated functionality we have:
>  and tools are recommended to continue supporting this for the time being

I mean, being deprecated should always be read as being required for tools to implement, but that library authors should stop using it as it is planned for removal in some future version of the specification.

Reply:
I added a reference to the section where `"builtin"` is used.